Content/Guides/syslog-ng-guide-admin/parser-regexp.htm

```diff
@@ -17,6 +17,7 @@ <h1 name="parser-regexp">Regular expression (regexp) parser</h1>
         <MadCap:keyword term="parsers:['Regular expression']">
         </MadCap:keyword>
         <p>The <MadCap:variable name="General.abbrev" /> application can parse fields from a message with the help of regular expressions. This can be also achieved with the <span class="Code">match()</span> filter, by setting the store-matches flag, but the <span class="Code">regexp-parser()</span> offers more flexibility, like multiple patterns and setting the prefix of the created name-value pairs.</p>
+        <p>Note that <span class="Code">regexp-parser()</span> can create additional name-value pairs only if &quot;named capture groups&quot; used in the regular expression. See &quot;named capture groups&quot; in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a> for more information.</p>
```

Comment:
Az ilyen note-okhoz van nekünk egy ilyen formázott Note tagünk, ebbe bele tudod tenni az egészet, légyszi? És akkor a "Note that" a szöveg elejére nem is kell majd.

`<p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span><span class="Code">  **IDE A SZÖVEG** </p></div>`

Reply:
Idézőjelet nem szoktunk használni, helyette lehet ez:  
`<b class="UI">`
mindkétszer, a **named question groups**-ra

Reply:
a **named capture groups**-t lehet nagybetűvel kezdeni ---> **Named capture groups**

Reply:
Az utolsó mondatban a szórendet kicsit fordísuk meg, legyen: **For more information, see ....**

Reply:
Fixed in #90